### DISCUSSION TITLE
Who voted?

### DISCUSSION THREAD

**Discussion by mktcode** [community/community#5650] [Discussions]
The [`Votable` interface](https://docs.github.com/en/graphql/reference/interfaces#votable) only has a vote counter and flags indicating whether the current viewer can or already has cast a vote. In an application I would like to have access to who voted on a discussion, e.g. to apply voting weights based on account information. Is this something to look out for in the future?

**Comment by gdvalderrama** [comment]
Came here to say exactly that: I can see which users reacted with an emoji, but I can't see which users voted.
It'd also be great to be able to filter discussions voted by a particular user.

**Comment by friday** [comment]
I think being able to upvote (and downvote if implemented) like Reddit without others being able to see your votes has a value in itself though.

I've been thinking to use a Discussions topic for user being able to contribute plugins to what the main GitHub repo deal with, and anonymous voting could be good if they are buggy, intrusive or bloated.

**Reply by mktcode** [reply]
Could be bound to repository access and of course having this as an optional feature would be best.
